apps/logo-quiz/src/main.tsx

```diff
@@ -20,35 +20,38 @@ import { FirebaseContext } from './shared/components/firebase/with-firebase';
 import { Firebase } from './shared/components/firebase/firebase';
 import { ROUTES } from './shared/utils/routes';
 import { environment } from '@logo-quiz/environment';
+import { ErrorBoundary } from './shared/components/error-boundary/error-boundary';
 
 const loggerMiddleware = createLogger();
 
 const middlewares = [
-  thunkMiddleware,
+  thunkMiddleware, //
```

Comment:
are the // necessary?

Reply:
Yeah. I'm using prettier and [it requires that nasty hack](https://github.com/prettier/prettier-vscode/issues/352#issuecomment-417471927) to put everything into a single line. It sucks, I know :/

Reply:
alright, good to know about this